test/model/tools/model_test/main.cpp

```diff
@@ -460,7 +460,7 @@ main(int argc, char *argv[])
 {
     model::kv_workload_generator_spec spec;
 
-    uint64_t base_seed = (uint64_t)time(NULL);
+    uint64_t base_seed = model::random::next_seed(time(NULL));
```

Comment:
I don't see how this solves the problem in the ticket.  If two processes running this test start at the same time, won't they get the same base_seed, and all subsequent seeds?

Reply:
The main issue was that if two Evergreen runs do 100 iterations and one starts at UNIX time 3000 and the other one at time 3002, they would each cover 98 of the same seeds: the first one would do 3000–3100 and the other one would do 3002–3102. The ticket fixed that through using `next_seed` when advancing the timestamp in the `for` loop at the end of the function. (I added the `next_seed` here to quickly fill out the higher-order bits that were not populated by `time` alone.)

But yes, there is a small chance of the two starting within the same second—I just added an XOR with RDTSC to be safe (and it's an XOR rather than just using it directly on the off chance it's being run on a platform that does not have the instruction).